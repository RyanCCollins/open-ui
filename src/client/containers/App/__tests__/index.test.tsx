import * as React from 'react';
import Container from '../';
import State from '../State';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import { Provider } from 'react-redux';
import mockStore from 'test/mockstore';

const testState: State = {
  isMobile: true,
  logoText: 'logo',
  navLinks: [{
    text: 'demo',
    url: 'www.google.com',
  }, {
    text: 'demo2',
    url: 'www.gmail.com',
  }],
};

describe('DocsContainer ', () => {
  let store: any;
  beforeEach(() => {
    store = mockStore(testState);
  });

  it('should render with default props', () => {
    const wrapper = shallow(
      <Provider store={store}>
        <Container />
      </Provider>,
    );
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
