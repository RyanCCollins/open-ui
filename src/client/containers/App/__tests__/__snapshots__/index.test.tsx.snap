// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DocsContainer  should render with default props 1`] = `<Connect(Feature) />`;
